server/controllers/request.js

```diff
@@ -100,22 +54,25 @@ exports.getRequest = asyncHandler(async (req, res, next) => {
 // @desc Update a user's request
 // @access Private
 exports.editRequest = asyncHandler(async (req, res, next) => {
+  console.log(req.body);
   const body = req.body;
-  const verifyUser = await Request.findOne({ _id: body.id });
+  const verifyUser = await Request.findOne({ _id: body._id });
 
   if (req.user.id != (verifyUser.ownerId || verifyUser.sitterId)) {
     res.status(401);
     throw new Error("Not authorized");
   }
   const updatedRequest = await Request.findOneAndUpdate(
-    { _id: body.id },
+    { _id: body._id },
     body,
     { new: true }
   );
   if (updatedRequest) {
+    let requests = await Request.find({ sitterId: req.user.id });
```

Comment:
Please use `const` keyword if you're not reassigning this variable.

Reply:
changed to const